avocado/utils/lv_utils.py

```diff
@@ -339,14 +339,25 @@ def lv_remove(vg_name, lv_name):
     process.run(cmd, sudo=True)
 
 
-def lv_create(vg_name, lv_name, lv_size, force_flag=True):
+def lv_create(vg_name, lv_name, lv_size,
```

Comment:
This can break current tests, even though the API may look worse than your proposal, please consider adding the new parameters to the end.  That is, `force_flag` should continue being the third parameter, so users using positional parameters won't be surprised by the order change.

Reply:
Basically this: https://github.com/avocado-framework/avocado/pull/2932/commits/72acba0121e0824d79b06e554093996e8bc79b30

Reply:
I see. I guess we have many users of this part of the code? This is fine of course, a small sacrifice to make for compatibility.